config/locales/en.yml

```diff
@@ -635,13 +635,37 @@ en:
   # Label for an option to show/include content by any user
   any_user: Any
   anyone: Anyone
+  app_owner_application_about_p1_html:  |
+    iNaturalist third party apps allow authenticated use of the iNaturalist API. 
+    This allows you to make API calls that do things like create observations or
+    identifications. Usual use cases are apps such as desktop apps or mobile apps
+    that include functionality for their user(s) to create iNaturalist observations. 
+    This content must be generated with real humans in the loop. Please read our
+    community guidelines prohibiting machine generated observations, identifications,
+    and comments for more details.
+  app_owner_application_about_p2_html:  |
+    In order to create third party apps, you must first submit an application to earn
+    the role of ‘app owner’. This application should describe the functionality your app
+    will have and how you plan to use it. In order to submit an application your account
+    must be at least 2 months old and must have made at least 10 improving identifications
+    a week for each of the last 4 consecutive weeks. We may ask followup questions about your
+    plan and we reserve the right to turn down applications for any reason.
```

Comment:
Here's a re-write with American usage consistent with the rest of the default English text of the site (double quotes, terminal quote right of period, hyphenated adjectival phrases):

> In order to create third-party apps, you must first submit an application to become an "app owner." This application should describe the functionality your app will have and how you plan to use it. In order to submit an application your account must be at least 2 months old and must have made at least 10 improving identifications a week for each of the last 4 consecutive weeks. We may ask follow-up questions about your plan and we reserve the right to turn down applications for any reason.